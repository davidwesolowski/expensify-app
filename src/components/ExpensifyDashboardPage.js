import React from 'react';
import ExpensesList from './ExpensesList';
import ExpensesListFilter from './ExpensesListFilter';

const ExpensifyDashboardPage = () => (
    <div>
        <ExpensesListFilter />
        <ExpensesList />
    </div>
);

export default ExpensifyDashboardPage;