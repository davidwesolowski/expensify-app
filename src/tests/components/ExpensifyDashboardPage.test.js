import React from 'react';
import { shallow } from 'enzyme';
import ExpensifyDashboardPage from '../../components/ExpensifyDashboardPage';

test('Should render ExpensifyDashboardPage correctly', () =>
{
    const wrapper = shallow(<ExpensifyDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});