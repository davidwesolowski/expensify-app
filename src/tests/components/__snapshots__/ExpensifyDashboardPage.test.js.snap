// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpensifyDashboardPage correctly 1`] = `
<div>
  <Connect(ExpensesListFilter) />
  <Connect(ExpensesList) />
</div>
`;
